.github/workflows/linux.yml

```diff
@@ -0,0 +1,37 @@
+name: Checks on Linux
+
+on: [pull_request]
+
+jobs:
+  Checks-on-linux:
+
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - run: echo "Job triggered by a ${{ github.event_name }} event on branch is ${{ github.ref }} in repository is ${{ github.repository }}, runner on ${{ runner.os }}"
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      # You can test your matrix by printing the current Python version
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      - name: Install dependencies
+        run: |
+         pip install -r requirements-selftests.txt
+      - name: Installing Avocado in develop mode
+        run: python3 setup.py develop --user
+      - name: Avocado version
+        run: avocado --version
+      - name: Make check
+        env:
+          AVOCADO_LOG_DEBUG: "yes"
+          AVOCADO_CHECK_LEVEL: "1"
+        run:  make check
```

Comment:
Instead of running the full check, this step could run `selftests/check.py --disable-static-checks` and make this job dependent on the original static check job.

This avoids unnecessary multiple runs of the static checks.

Reply:
Thanks willian, make check is what we're currently using in travis.

Reply:
Right. If we take the opportunity to do some small reorg, that would be good too, but I'm fine with migration then reorg to be on the safe side.